src/Cryptol/ModuleSystem/Renamer.hs

```diff
@@ -887,7 +889,8 @@ patternEnv  = go
                 n   <- liftSupply (mkParameter (getIdent pn) loc)
                 return (singletonT pn n)
 
-  typeEnv (TRecord fs)      = bindTypes (map value fs)
+  typeEnv (TRecord fs)      = bindTypes (map (snd.snd) (canonicalFields fs))
```

Comment:
If this `.` is used as a function composition operator, then it's probably best for human parseability to put spaces around it